### PR TITLE
[WIP] Update upstream with medik8s consumers

### PR DESCRIPTION
**Pull request by prabinovRedhat**
Jira link for the epic - [ECOPROJECT-1466](https://issues.redhat.com/browse/ECOPROJECT-1466)
-----

We'd like to increase exposure and trust in our upstream community.

In order to do that we plan to  publish (after having proper permissions) medik8s consumers (End Users) :

in our [upstream website](https://www.medik8s.io/).
[Here](https://kubevirt.io/) is an example of how it was done for KubeVirt
in our [git repo ](https://github.com/medik8s), see an example [Here](https://github.com/kubevirt/kubevirt/blob/main/ADOPTERS.md) of how it was done for KubeVirt

------

### CODE REVIEW

**Comment by prabinovRedhat** [comment]
@mshitrit

**Comment by mshitrit** [comment]
@k-keiichi-rh
@mzeevi 
Can you please review ?

**Comment by clobrano** [comment]
Note that we need #61 before being able to test this locally

**Comment by mshitrit** [comment]
Hi @batzionb we will also appreciate you feedback on that as a UI expert :smile:

**Comment by batzionb** [comment]
I think a more standard approach is to use a [css grid](https://www.w3schools.com/css/css_grid.asp) to accomplish multiple rows of images with the desired layout

**Comment by batzionb** [comment]
I think it'll look much better just one row with three images, there's no need to take too much space just for three images
Then you also need less css and can make it more simple
Also worth checking if Jekyl has a standard way to embed this in Markdown instead of adding custom html and css

**Comment by prabinovRedhat** [comment]
I fixed according to @batzionb comment. As well I added NEC logo after their confirmation.

Screenshot 2023-09-26 at 16 43 27

File: end_users_section.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<div class="section-title" style="text-align: center; margin-top: 120px; margin-bottom: 60px; display: flex; justify-content: center; align-items: center;">
+    <h2>End Users</h2>
+  </div>
+  
+  <div class="logos-container" style="display: flex; flex-direction: row; justify-content: center; align-items: center; margin-bottom: 30px;">
```
Comment (batzionb) [inline end_users_section.html]: Since the styles are defined inline seems to me like you can remove the class attributes

Comment (prabinovRedhat) [reply end_users_section.html]: fixed

**Comment by prabinovRedhat** [comment]
sorry for inconvenience , I ran into some problem with my previous PR, I opened new one with all changes.
https://github.com/medik8s/docs/pull/64

cc: @mshitrit

**Pull request closed without merging**